src/main/java/ch/njol/skript/lang/ExperimentalSyntax.java

```diff
@@ -0,0 +1,20 @@
+package ch.njol.skript.lang;
+
+import org.skriptlang.skript.lang.experiment.Experiment;
+import org.skriptlang.skript.lang.experiment.ExperimentSet;
+
+/**
+ * A syntax element that requires an experimental feature to be enabled.
+ */
+public interface ExperimentalSyntax extends SyntaxElement {
+
+
```

Comment:
Ultimately up to the team but maybe include this as part of the `org.skriptlang.skript` package? Experiments are derived from there already and would make more sense there at least in my opinion.

Reply:
Yes, this should probably be in the lang.experiment package over there